binderhub/tests/test_build.py

```diff
@@ -215,22 +215,19 @@ def test_git_credentials_passed_to_podspec_upon_submit():
     assert env["GIT_CREDENTIAL_ENV"] == git_credentials
 
 
-async def test_local_repo2docker_build():
+async def test_local_repo2docker_build(io_loop):
```

Comment:
I don't think io_loop fixture should be needed here to pass explicitly to the Build. The build object should be able to manage its own reference to the calling loop if/when it needs it.